internal/operator-controller/applier/helm_test.go

```diff
@@ -280,11 +293,16 @@ func TestApply_InstallationWithPreflightPermissionsEnabled(t *testing.T) {
 		mockAcg := &mockActionGetter{
 			getClientErr: driver.ErrReleaseNotFound,
 			installErr:   errors.New("failed installing chart"),
+			desiredRel: &release.Release{
```

Comment:
> featuregatetesting.SetFeatureGateDuringTest(t, features.OperatorControllerFeatureGate, features.PreflightPermissions, true)

In line 273 is no longer necessary because the helm applier logic is based directly on the whether `PreAuthorizer` is non-nil.

Reply:
removed 273 and one other instance of that featuregate setup in the tests

Reply:
yep this is done

Reply:
I think `SingleOwnNamespaceInstallSupport` feature gate directly affects only the determination of the `watchNamespace`. The `PreAuthorizer` logic runs later and independently checks RBAC permissions based on the rendered Helm manifest, right? Putting this back fixes the pre-existing unit test.